### PR TITLE
Add new explore/observability related features to whats new docs

### PR DESCRIPTION
**Pull request by zoltanbedi**

### CODE REVIEW

**Comment by torkelo** [comment]
dont need to cherry pick doc updates :)

File: docs/sources/guides/whats-new-in-v7-2.md [hunk 1]
```diff
@@ -23,6 +23,9 @@ The main highlights are:
 - [**Table column filters**]({{< relref "#table-column-filters" >}})
 - [**New and improved transforms**]({{< relref "#new-and-improved-transforms" >}})
 - [**Field override matchers**]({{< relref "#field-override-matchers" >}})
+- [**Query inspector in Explore**]({{< relref "#query-inspector-in-explore" >}})
```
Comment (oddlittlebird) [inline docs/sources/guides/whats-new-in-v7-2.md:26]: Just a reminder, every feature should have a link to updated documentation.

File: docs/sources/guides/whats-new-in-v7-2.md [hunk 2]
```diff
@@ -77,6 +80,20 @@ The UI for transforms also has an update that now allows you to move transformat
 
 You can now add override rules that use a regex matcher and overrides that match on field type.
 
+## Query inspector in Explore
+
+You can enjoy all the details query inspector gave you in dashboards now in explore as well. You can open query inspector tab with the button next to query history.
```
Comment (oddlittlebird) [inline docs/sources/guides/whats-new-in-v7-2.md:85]: Capitalize the name of the tab. Match the UI.

File: docs/sources/guides/whats-new-in-v7-2.md [hunk 3]
```diff
@@ -77,6 +80,20 @@ The UI for transforms also has an update that now allows you to move transformat
...
+
+## $__rate_interval for Prometheus
+
+You can now use the new variable `$__rate_interval` in Prometheus for rate functions mainly. `$__rate_interval` in general is one scrape interval larger than `$__interval` but is never smaller than four times the scrape interval (which is 15s by default). See the [Prometheus datasource]({{< relref "../features/datasources/prometheus.md#using-__rate_interval-variable" >}}) for more details.
```
Comment (oddlittlebird) [inline docs/sources/guides/whats-new-in-v7-2.md:89]:
```suggestion
You can now use the new variable `$__rate_interval` in Prometheus for rate functions mainly. `$__rate_interval` in general is one scrape interval larger than `$__interval` but is never smaller than four times the scrape interval (which is 15s by default). See the [Prometheus data source]({{< relref "../features/datasources/prometheus.md#using-__rate_interval-variable" >}}) for more details.
```

Comment (oddlittlebird) [reply docs/sources/guides/whats-new-in-v7-2.md:89]: Per the Grafana style guide, "data source" is always two words.

File: docs/sources/guides/whats-new-in-v7-2.md [hunk 4]
```diff
@@ -77,6 +80,20 @@ The UI for transforms also has an update that now allows you to move transformat
...
+
+## Loki toggling parsed fields
+
+With this awesome contribution from one of our community members you can now toggle parsed fields in Explore.
```
Comment (oddlittlebird) [inline docs/sources/guides/whats-new-in-v7-2.md:93]:
```suggestion
With this awesome contribution from one of our community members, you can now toggle parsed fields in Explore.
```

**Review by oddlittlebird — CHANGES_REQUESTED** [review]

File: docs/sources/guides/whats-new-in-v7-2.md (hunk 2 above)
Comment (zoltanbedi) [reply docs/sources/guides/whats-new-in-v7-2.md]: Done

File: docs/sources/guides/whats-new-in-v7-2.md (hunk 1 above)
Comment (zoltanbedi) [reply docs/sources/guides/whats-new-in-v7-2.md:26]: Added

File: docs/sources/guides/whats-new-in-v7-2.md [hunk 5]
```diff
@@ -77,6 +80,20 @@ The UI for transforms also has an update that now allows you to move transformat
...
 
+## Query inspector in Explore
+
+You can enjoy all the details query inspector gave you in dashboards now in Explore as well. You can open query inspector tab with the button next to query history. See [Query inspector in Explore]({{< relref "../features/explore/index.md#query-inspector" >}}) for more details.
+
+## $__rate_interval for Prometheus
+
+You can now use the new variable `$__rate_interval` in Prometheus for rate functions mainly. `$__rate_interval` in general is one scrape interval larger than `$__interval` but is never smaller than four times the scrape interval (which is 15s by default). See the [Prometheus data source]({{< relref "../features/datasources/prometheus.md#using-__rate_interval-variable" >}}) for more details.
+
+## Loki toggling parsed fields
```
Comment (oddlittlebird) [inline docs/sources/guides/whats-new-in-v7-2.md:91]:
```suggestion
## Loki now toggles parsed fields
```

File: docs/sources/guides/whats-new-in-v7-2.md [hunk 6]
```diff
@@ -77,6 +80,20 @@ The UI for transforms also has an update that now allows you to move transformat
 
 You can now add override rules that use a regex matcher and overrides that match on field type.
 
+## Query inspector in Explore
```
Comment (oddlittlebird) [inline docs/sources/guides/whats-new-in-v7-2.md:83]:
```suggestion
## Inspect queries in Explore
```

**Review by oddlittlebird — APPROVED** [review]
Two minor suggestions, then good to go!

File: docs/sources/guides/whats-new-in-v7-2.md (hunk 5 above)
Comment (davkal) [reply docs/sources/guides/whats-new-in-v7-2.md:91]: It's not Loki toggling the fields. The user can choose which parsed fields should be shown. When a user toggles a parsed field so it's shown, the other fields are being hidden.

Comment (zoltanbedi) [reply docs/sources/guides/whats-new-in-v7-2.md:91]: @oddlittlebird can you help me find a better title for this feature?

Comment (oddlittlebird) [reply docs/sources/guides/whats-new-in-v7-2.md:91]: Toggle parsed Loki fields?

Comment (zoltanbedi) [reply docs/sources/guides/whats-new-in-v7-2.md:91]: Sounds good to me. Let me change that.

**Pull request merged**